test/models/diffusion/test_song_unet_pos_embd.py

```diff
@@ -0,0 +1,231 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
needs test where global_index is passed.

Reply:
added tests